pkg/config/context.go

```diff
@@ -86,4 +87,9 @@ var ContextOptions = []ContextOption{
 		Description: "Specifies the timeout to inject the agent",
 		Default:     "20",
 	},
+	{
+		Name:        ContextOptionRegistryCache,
+		Description: "Specifies the registry to use as a build cache",
```

Comment:
In what format does it need to be passed in?

Reply:
good point, I'll update